forge-game/src/main/java/forge/game/ability/effects/ManaReflectedEffect.java

```diff
@@ -23,16 +23,14 @@ public class ManaReflectedEffect extends SpellAbilityEffect {
     public void resolve(SpellAbility sa) {
         // Spells are not undoable
         AbilityManaPart ma = sa.getManaPart();
-        sa.setUndoable(sa.isAbility() && sa.isUndoable());
+        sa.setUndoable(sa.isAbility() && sa.isUndoable() && sa.getSubAbility() == null);
```

Comment:
what if it had two+ mana abilities as chain?
Like add Mana, and then add Additional Mana?

or doesn't this matter for Mana Reflected?

Reply:
Couldn't find any like that
also ManaEffect will do the same for subs (in `GameActionUtil.generatedMana`)